src/components/TopBar/TopBar.vue

```diff
@@ -58,6 +58,19 @@
 		<CallTime v-if="isInCall"
 			:start="conversation.callStartTime" />
 
+		<!-- Participants counter -->
+		<NcButton v-if="isInCall && !isOneToOneConversation"
+			v-tooltip="t('spreed', 'Participants in call')"
+			class="top-bar__button"
+			type="tertiary"
+			@click="openSidebar('participants')">
```

Comment:
Tab is not available for guests.
Is this gracefully handled?

Reply:
I'll check it. Should the counter be visible for guests in that case?

Reply:
I leave this up to you